services/provider/server/server.go

```diff
@@ -644,26 +649,30 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 				"csi.storage.k8s.io/controller-expand-secret-name": provisionerSecretName,
 			}
 
-			extR = append(extR, &pb.ExternalResource{
-				Name: "cephfs",
-				Kind: "StorageClass",
-				Data: mustMarshal(cephfsStorageClassData),
-			})
-
-			extR = append(extR, &pb.ExternalResource{
-				Name: cephRes.Name,
-				Kind: cephRes.Kind,
-				Data: mustMarshal(map[string]string{
-					"filesystemName": subVolumeGroup.Spec.FilesystemName,
-				})})
-
-			extR = append(extR, &pb.ExternalResource{
-				Name: "cephfs",
-				Kind: "VolumeSnapshotClass",
-				Data: mustMarshal(map[string]string{
-					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
-					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
-				})})
+			extR = append(extR,
+				&pb.ExternalResource{
+					Name: "cephfs",
+					Kind: "StorageClass",
+					Data: mustMarshal(cephfsStorageClassData),
+				},
+				&pb.ExternalResource{
+					Name: cephRes.Name,
+					Kind: cephRes.Kind,
+					Data: mustMarshal(map[string]string{
+						"filesystemName": subVolumeGroup.Spec.FilesystemName,
+					})},
+				&pb.ExternalResource{
+					Name: "cephfs",
+					Kind: "VolumeSnapshotClass",
+					Data: mustMarshal(map[string]string{
+						"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
+					})},
+				&pb.ExternalResource{
+					Name: "cephfs",
+					Kind: "VolumeGroupSnapshotClass",
+					Data: mustMarshal(map[string]string{
+						"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
```

Comment:
https://github.com/kubernetes-csi/external-snapshotter/blob/c72f087f751abb285a90c4f5bb02df9014d2bc19/pkg/utils/util.go#L62-L63 is the key for groupsnapshot secret